x-pack/plugins/observability_ai_assistant/scripts/evaluation/kibana_client.ts

```diff
@@ -39,6 +39,8 @@ interface ChatClient {
     {}: { conversationId?: string; messages: InnerMessage[] },
     criteria: string[]
   ) => Promise<EvaluationResult>;
+  getResults: () => EvaluationResult[];
+  onResult: (cb: (result: EvaluationResult) => void) => () => void;
```

Comment:
Could you add these methods to `x-pack/plugins/observability_ai_assistant/public/types.ts` and the mock of ChatClient in `x-pack/plugins/observability_ai_assistant/public/mock.tsx`?

I'm surprised the TS check did not catch this..

Reply:
Different chat client, this is only for the tests

Reply:
Oh wait, this is a different ChatClient in `/scripts`. Should they have the same methods?

Reply:
It's a simplified client so it's easy to use for Almu & Marco as well, so I don't think we should have parity with the other client as a goal necessarily